client/app/widgets/Resources/index.jsx

```diff
@@ -110,28 +118,53 @@ export class Resources extends React.Component<Props, State> {
     });
   };
 
+  onClick = () => {
+    const { resources } = this.props;
+    this.setState((prevState: State) => ({
+      resourcesDisplayed: prevState.resourcesDisplayed + RESOURCES_PER_PAGE,
+      lastPage:
+        prevState.resourcesDisplayed + RESOURCES_PER_PAGE >= resources.length,
+    }));
+  };
+
+  displayLoadMoreButton = () => (
```

Comment:
As outlined by Codeclimate in this [error](https://codeclimate.com/github/ifmeorg/ifme/pull/1561), this is component is identical to the one used in `client/app/components/BaseContainer/index.jsx`. I recommend extract both as a generic component called `< LoadMoreButton />` and using it in both places.

Reply:
We also want to write integration tests for this!